automated-demo/deploy-haproxy.md

```diff
@@ -16,7 +16,7 @@
 4. Create a namespace/use an existing one and create the configmap from our definition
 
   ```sh
-  oc -n <your_namespace> create configmap haproxy --from-file=yaml-resources/haproxy/haproxy
+  oc -n <your_namespace> create configmap haproxy --from-file=yaml-resources/haproxy/haproxy.tmpl
```

Comment:
The .tmpl file is supposed to be copied. The idea here is preventing changes to this file to hit the git repo, this file is allowed to be committed in the .gitignore. Maybe we can update the instructions to include the copy command.